code/world.dm

```diff
@@ -56,6 +57,19 @@
 			external_rsc_urls.Cut(i,i+1)
 #endif
 
+/world/proc/CheckSchemaVersion()
+	if(config.sql_enabled)
+		if(SSdbcore.Connect())
```

Comment:
If you're running this before `SetRoundID`, thus making it the first attempt to connect to the DB, move these two messages to this proc: https://github.com/tgstation/tgstation/blob/master/code/world.dm#L62 and https://github.com/tgstation/tgstation/blob/master/code/world.dm#L70